lib/game/widget_builder.dart

```diff
@@ -231,11 +231,19 @@ class WidgetBuilder {
       ),
     );
 
-    if (_hasBasicGestureDetectors(game)) {
+    final hasBasicDetectors = _hasBasicGestureDetectors(game);
+    final hasAdvancedDetectors = _hasAdvancedGesturesDetectors(game);
+
+    assert(
+      !(hasBasicDetectors && hasAdvancedDetectors),
+      'WARNING: Both Basic and Advanced detectors detected. Advanced detectors will override basic detectors and the later will not receive events',
+    );
```

Comment:
nit: `Both Basic and Advanced detectors` => `Both Advanced and Basic detectors ` might be more clear because that is the order you use on the second sentence with later